docs/chapter2-payload-image-format.rst

```diff
@@ -250,7 +250,7 @@ Nodes with properties are shown in the tables below:
    uncomp-size       OR          u32          Size of the uncompressed data in bytes. If
                                               the data is not compressed, this can be
                                               omitted.
-   entry-start       O           u32 / u64    If required, this is the offset of the image
+   entry             O           u32 / u64    If required, this is the offset of the image
```

Comment:
Basing on Benjamin's finding, this is absolute address, not offset to load address. Should we update this description?

Reply:
done.